prac_05/word_occurrences.py

```diff
@@ -0,0 +1,18 @@
+
+user_words = input("Please enter your statement: ")  # Step 1
+print(user_words)  # Step 1
+count_of_words = dict()  # Step 2 - Convert to Dictionary
+words = user_words.split()  # Step 3 - Splitting string into 'words', storing in 'words'
+
+for every_word in words:  # Step 4 - Count the frequency of words in "words"
+    if every_word in count_of_words:
+        count_of_words[every_word] += 1  # If the words is coming twice, iterate
+    else:
+        count_of_words[every_word] = 1
+data_of_words = []  # Step 5 - declare empty list to store 2 things - Words - & their respective frequencies
+for word_name, word_frequency in count_of_words.items():
```

Comment:
This doesn't seem helpful. Why iterate through items and then not use word_frequency? You can just iterate through the keys if you don't need the value. Plus, this just gets you a list of the keys, which you can do in one statement (see solutions).

Reply:
I just realised I'm commenting on prac 5 work in your prac 8 pull request. See if you can learn how to create PRs/branches that only contain new/desired work.

Reply:
Sure Lindsay. Thanks much.